packages/gatsby-link/package.json

```diff
@@ -17,8 +17,8 @@
   "scripts": {
     "build": "npm-run-all --npm-path npm -s build:cjs build:esm",
     "postbuild": "prepend-directive --files=dist/index.js,dist/index.modern.mjs --directive=\"use client\"",
-    "build:cjs": "microbundle -f cjs --jsx React.createElement --generateTypes false  -i src/index-cjs.js -o dist/index.js",
-    "build:esm": "microbundle -f modern --jsx React.createElement --generateTypes false -o dist/index.mjs",
+    "build:cjs": "microbundle -f cjs --jsx React.createElement --generateTypes false  -i src/index.js -o dist/index.js",
```

Comment:
Should we be able to completely revert the package.json change, too?
https://github.com/gatsbyjs/gatsby/pull/36312/files#diff-d7325f88747732a50f1e5f640668bc720d7cb71eab3b349894be81132bedb602

The former should work fine now

Reply:
Done in https://github.com/gatsbyjs/gatsby/pull/36888/commits/b28c0c338c8de58f722bd585445f412a6fdeb5e3